rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/health/10_basic.yml

```diff
@@ -10,11 +10,4 @@
   - is_true: cluster_name
   - match:   { status: "GREEN" }
   - match:   { impacts: [] }
-  - match:   { components.cluster_coordination.status: "GREEN" }
-  - match:   { components.cluster_coordination.indicators.instance_has_master.status: "GREEN" }
-  - match:   { components.cluster_coordination.indicators.instance_has_master.summary: "Health coordinating instance has a master node." }
-  - is_true: components.cluster_coordination.indicators.instance_has_master.details.coordinating_node.node_id
-  - is_true: components.cluster_coordination.indicators.instance_has_master.details.coordinating_node.name
-  - is_true: components.cluster_coordination.indicators.instance_has_master.details.master_node.node_id
-  - is_true: components.cluster_coordination.indicators.instance_has_master.details.master_node.name
-  - match:   { components.snapshots.status: "GREEN" }
```

Comment:
Could we instead ensure the `ClusterCoordination` health indicator is included too? That would demonstrate how to add indicators in the server module.